saul-core/doc/SAULLANGUAGE.md

```diff
@@ -17,6 +17,18 @@ object OrgClassifier extends Learnable[ConllRawToken](ErDataModelExample) {
     containsInPersonList, wordLen, containsInCityList)
 }
 ```
+### train and test classifiers
+
+call `train()` method to train your classifier using the populated data in the data model's training instances:
+
+```scala
+OrgClassifier.train()
+```
+call `test()` method to test your classifier using the populated data model's test instance:
+
+ ```scala
+ OrgClassifier.test()
+```
```

Comment:
Should we or we shouldn't explain the other possible arguments for the `test` function? 
https://github.com/IllinoisCogComp/saul/blob/master/saul-core/src/main/scala/edu/illinois/cs/cogcomp/saul/classifier/Learnable.scala#L289-L318